test/test_suit/test_avro_avro.py

```diff
@@ -31,7 +31,7 @@ def verify(self, round):
         # validate content of line 1
         res = self.driver.snowflake_conn.cursor().execute(
             "Select * from {} limit 1".format(self.topic)).fetchone()
-        goldMeta = r'{"CreateTime":\d*,"key":[{"timestamp":\d*,"tweet":"Rock:Nerfpaper,scissorsisfine.",' \
+        goldMeta = r'{"CreateTime":\d*,"SnowflakeConnectorTime":\d*,"key":[{"timestamp":\d*,"tweet":"Rock:Nerfpaper,scissorsisfine.",' \
```

Comment:
is the default set to true? this seems like a breaking change?

Reply:
+1, default should be false. Also the name doesn't seem to much with above

Reply:
Turned the parameter `false` by default. Yes, the property name in the test had not been updated.

Reply:
We can have client breaking changes. It just needs to be documented. 
I don't think having it default true is a bad thing

Reply:
Shouldn't the connector follow BCR too?
If not then both options sound alright

Reply:
Client BCRs are different than Snowflake server release BCRs. Yes, there is still approval and review but to customers, there is a notice and customers can choose to upgrade to the newest client version. 
I'll forward the internal doc.